examples/lib/defaultHtmlStepUi.js

```diff
@@ -68,18 +66,18 @@ function DefaultHtmlStepUi(_sequencer, options) {
     var util = intermediateHtmlStepUi(_sequencer, step);
 
     var parser = new DOMParser();
-    step.ui = parser.parseFromString(step.ui, 'text/html');
+    step.ui = parser.parseFromString(step.ui, 'text/html'); // convert the markup string to a DOM node
     step.ui = step.ui.querySelector('div.container-fluid');
 
-    $step = scopeQuery.scopeSelector(step.ui);
+    $step = scopeQuery.scopeSelector(step.ui); // Shorthand methods for scoped DOM queries. Read the docs(CONTRIBUTING.md) for more info
     $stepAll = scopeQuery.scopeSelectorAll(step.ui);
     step.ui.$step = $step;
     step.ui.$stepAll = $stepAll;
 
-    step.linkElements = step.ui.querySelectorAll('a');
-    step.imgElement = $step('a img.img-thumbnail')[0];
+    step.linkElements = step.ui.querySelectorAll('a'); // all the anchor tags in the step UI
```

Comment:
Can we have consistency among comments, please?
Like, use capital case letters while beginning a comment. Using a full stop after the end. Write short descriptive sentences. Using 3rd person passive voice.

Reply:
I'll do that. Ty for pointing that out.